.github/workflows/deploy.yml

```diff
@@ -2,14 +2,20 @@ name: Deploy Styleguide
 
 on:
   push:
-    branches:
-      - master
+#    branches:
+#      - master
 
 jobs:
   build:
     name: Build and Deploy
     runs-on: ubuntu-latest
     steps:
+      - name: Setup fontawesome dependencies
+          run: |
```

Comment:
I'm not sure why it's upset: `You have an error in your yaml syntax on line 14`

Reply:
`You have an error in your yaml syntax on line 22` I'm guessing it's a tabs / spaces nuanced issue

Reply:
uyeah.. ugh